apps/nextra/pages/en/build/create-aptos-dapp/templates/digital-asset.mdx

```diff
@@ -0,0 +1,220 @@
+
+1. Run `npm run move:init` - a command to initialize an account to publish the Move contract. When you run that command it will:
+```sh filename="Terminal"
+npm run move:init
+```
+   - Generate a new CLI `.aptos/config.yaml` file that holds a profile with the account private key, account address, and network configuration.
+   - Configure your development environment and add a `VITE_MODULE_ADDRESS` variable into the `.env` file with the account address from the previous step.
+
+2. To set who can create a new collection, edit the `.env` file and update `VITE_CREATOR_ADDRESS` to be the address of the account allowed to create collections.
+
+3. Run
+```sh filename="Terminal"
+npm run move:publish
+```
+
+### Connect a wallet
+
+Once you have [published the move module](#publish-the-move-module), you will need to connect your wallet to submit transactions. Make sure your Wallet is set to the same account you used in the [previous section](#publish-the-move-module) to publish your Move module.
+Also ensure that your wallet is set to the same network you selected to work with (e.g. testnet).
+
+Now, you can connect your wallet by clicking on the "Connect Wallet" button at the top right, and start with creating your NFT collection.
+
+### Upload Collection Files Data
+
+To create a NFT collection you will need image files and metadata files.
+
+Image files can be any of the type - `"png", "jpg", "jpeg", "gif"`. But, make sure all image files have the same file extension. If some images have `jpeg` extension and some `jpg` extension, the upload will fail.
+
+Metadata files are `json` files that hold the Collection data and each of the NFT data.
+
+`create-aptos-dapp` requires you to upload a folder with all the relevant files, a folder structure should be
+
+
+<FileTree>
+  <FileTree.Folder name="assets" defaultOpen>
+    <FileTree.File name="collection.jpeg" />
+    <FileTree.File name="collection.json" />
+    <FileTree.Folder name="images">
+      <FileTree.File name="1.jpeg" />
+      <FileTree.File name="2.jpeg" />
+      <FileTree.File name="3.jpeg" />
+    </FileTree.Folder>
+    <FileTree.Folder name="metadatas">
+      <FileTree.File name="1.json" />
+      <FileTree.File name="2.json" />
+      <FileTree.File name="3.json" />
+    </FileTree.Folder>
+  </FileTree.Folder>
+</FileTree>
+
+<Callout type="info">
+  Make sure all image files have the same file extension. If some images will
+  have `jpeg` extension and some `jpg` extension, the upload will fail.
+</Callout>
+
+{ /* TODO: change ERC-1155 once finish rewriting the NFT standards page */ }
+
+We recommend the developers follow the [ERC-1155 off-chain data](https://eips.ethereum.org/EIPS/eip-1155) schema to format their JSON files. One thing to note, since the image field will be updated with the Arweave link after this step, feel free to leave it empty.
+
+Example of `collection.json` file
+
+```js filename="collection.json"
```

Comment:
```suggestion
```json filename="collection.json"
```

Reply:
This one is a bit messy because of the comments, so I made it javascript.